tests/migration_test.go

```diff
@@ -592,29 +592,53 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		Expect(suiteConfig.ParallelTotal).To(Equal(1), "this test is supported for serial tests only")
 	}
 
-	expectEvents := func(eventListOpts metav1.ListOptions, expectedEventsAmount int) {
+	expectEventsAmount := func(eventListOpts metav1.ListOptions, expectedEventsAmount int, namespace string) *k8sv1.EventList {
 		// This function is dangerous to use from parallel tests as events might override each other.
 		// This can be removed in the future if these functions are used with great caution.
 		expectSerialRun()
+		var events *k8sv1.EventList
 
 		Eventually(func() []k8sv1.Event {
-			events, err := virtClient.CoreV1().Events(util.NamespaceTestDefault).List(context.Background(), eventListOpts)
+			events, err = virtClient.CoreV1().Events(namespace).List(context.Background(), eventListOpts)
 			Expect(err).ToNot(HaveOccurred())
 
 			return events.Items
 		}, 30*time.Second, 1*time.Second).Should(HaveLen(expectedEventsAmount))
+
+		return events
 	}
 
-	deleteEvents := func(eventListOpts metav1.ListOptions) {
-		// See comment in expectEvents() for more info on why that's needed.
+	expectAtLeastOneEvent := func(eventListOpts metav1.ListOptions, namespace string) *k8sv1.EventList {
```

Comment:
`expectAtLeastOneEvent` and `expectEventsAmount` are very similar. Any chance to merge them in one more general function? Thanks

Reply:
i moved the functional tests to another file ,
They aren't in the same package now.
so i kept migration_test.go without any change.